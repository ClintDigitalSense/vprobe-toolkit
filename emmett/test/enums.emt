# verify that enum ordering works the way it ought to.
enum { A = 12, B = 3, C };
VMM1Hz printf("%d, %d, %d\n", A, B, C);
