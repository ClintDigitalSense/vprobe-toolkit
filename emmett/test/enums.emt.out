(version 0.2)

 ;;;;;
(vprobe VMM1Hz ( printf    "%d, %d, %d\n"  12  3  4 ))